auth/examples_test.go

```diff
@@ -2,11 +2,14 @@ package grpc_auth_test
 
 import (
 	"context"
+	"log"
+
+	grpc_auth "github.com/grpc-ecosystem/go-grpc-middleware/auth"
+	grpc_ctxtags "github.com/grpc-ecosystem/go-grpc-middleware/tags"
```

Comment:
Can you put this import group underneath the third party imports, or together. It doesn't matter too much (we've fixed it in v2).